### PR TITLE
Aggregate stdlib under core so it gets published along with others

### PR DESCRIPTION
**Pull request by rexim**
Everything is already published to local repo using these changes. The request to link local packages to JCenter is already sent. The corresponding issue will be closed once the request is approved or something.

Related to #325

### CODE REVIEW

**Review by ForNeVeR — APPROVED** [review]

**Comment by rexim** [comment]
@ForNeVeR thanks for the review!

**Comment by keddelzz** [comment]
I approve the changes, too!

Sorry for the late answer, I've been on vacation this weekend.

**Comment by rexim** [comment]
@keddelzz, don't even apologize. No one's obligated here. But opinion of the core developers is always valuable. Even after the merge. Thanks for spending your time on the review. :)

**Pull request merged**
